tests/features/project.feature

```diff
@@ -45,9 +46,10 @@ Feature: Project content creation
     And I should not see "Committee on Agriculture and Rural Development"
     And I should see "European Patent Office"
     And I should see "Audit Board of the European Communities"
+    And I should see "Anti Fraud Information System (AFIS)"
     And I should not see "Project teaser"
     And I should see "Summary text"
     And I should see "Body text"
     And I should see "2019-02-23"
     And I should see "Website"
-    And I should see "Example proposal"
+    And I should see "Example proposal"
```

Comment:
Restore the empty line in the end of the file.

Reply:
Fixed